web/src/pages/login.tsx

```diff
@@ -0,0 +1,61 @@
+import { JSX, Show, createResource, createSignal } from 'solid-js';
+import { isLoggedIn, logIn } from '../api';
+
+interface LoginPageOrProps {
+  children: JSX.Element;
+}
+export const LoginPageOr = (props: LoginPageOrProps) => {
+  return (
+    <>
+      <Show when={isLoggedIn()}>{props.children}</Show>
```

Comment:
We can get away with only one `<Show />` when we use the `fallback` prop.

Reply:
917dc69eb1c5f37890c2843c727e2ce532786c2a